browser/extensions/brave_extension_functional_test.cc

```diff
@@ -39,6 +40,7 @@ void ExtensionFunctionalTest::InstallExtensionSilently(ExtensionService* service
 
   const Extension* extension = registry_observer.WaitForExtensionReady();
   EXPECT_TRUE(extension);
+  return extension;
```

Comment:
I return this just so I can log the generated extension ID more easily in debugging.